system/lib/libcxxabi/src/cxa_personality.cpp

```diff
@@ -981,16 +988,6 @@ __gxx_personality_v0
       exception_header->catchTemp = 0;
 #endif
     }
-#ifdef __USING_WASM_EXCEPTIONS__
-    // Wasm uses only one phase in _UA_CLEANUP_PHASE, so we should set
-    // these here.
-    __cxa_exception* exception_header = (__cxa_exception*)(unwind_exception+1) - 1;
-    exception_header->handlerSwitchValue = static_cast<int>(results.ttypeIndex);
-    exception_header->actionRecord = results.actionRecord;
-    exception_header->languageSpecificData = results.languageSpecificData;
-    exception_header->catchTemp = reinterpret_cast<void*>(results.landingPad);
-    exception_header->adjustedPtr = results.adjustedPtr;
-#endif
```

Comment:
This is what the upstream code does in the first search phase, but we used only the cleanup phase, we duplicated the code here. Now we can remove it.